decidim-age_action_authorization/lib/decidim/age_action_authorization/authorizer.rb

```diff
@@ -22,6 +29,13 @@ def valid_age?
         (birthdate + minimum_age.years) <= Date.current
       end
 
+      def valid_subcensus?
+        Decidim::Census::SubcensusDocument.where(
```

Comment:
I would change this condition to make it optional to use subcensus. The way you've developed you always need to create a subcensus for every participatory process.

```ruby
def valid_subcensus?
   return true unless Decidim::Census::SubcensusDocument.where(decidim_census_subcensus_id: @component&.participatory_space&.id).any?

   Decidim::Census::SubcensusDocument.where(id_document: @authorization&.unique_id, decidim_census_subcensus_id: @component&.participatory_space&.id).any?
end
```